sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -97,7 +97,6 @@ public TextAnalyticsServiceVersion getServiceVersion() {
     }
 
     Mono<Response<DetectLanguageResult>> detectLanguageWithResponse(String text, String countryHint, Context context) {
-        Objects.requireNonNull(text, "'text' cannot be null.");
```

Comment:
what reason to remove requireNonNull check for `text` input

Reply:
On the client-side we should be doing as fewer validations as possible as service side is already doing those.